### PR TITLE
Allow multiple rules

### PR DESCRIPTION
**Pull request by fclaeys**
This PR allow to set more than one route in labels/tags (#419)

### CODE REVIEW

File: docs/basics.md [hunk 1]
```diff
@@ -87,13 +89,13 @@ Here is an example of frontends definition:
...
   backend = "backend2"
-    rule = "Path:/test"
+    rule = "Host: test3.localhost;Path:/test"
 ```
 
 - Three frontends are defined: `frontend1`, `frontend2` and `frontend3`
 - `frontend1` will forward the traffic to the `backend2` if the rule `Host: test.localhost, test2.localhost` is matched
 - `frontend2` will forward the traffic to the `backend1` if the rule `Host: localhost, {subdomain:[a-z]+}.localhost` is matched (forwarding client `Host` header to the backend)
-- `frontend3` will forward the traffic to the `backend2` if the rule `Path:/test` is matched
+- `frontend3` will forward the traffic to the `backend2` if the rule `Host: test3.localhost; Path:/test` is matched
```
Comment (emilevauge) [inline docs/basics.md]: > if the rule `Host: test3.localhost; Path:/test` is matched

I would prefer: if the rules `Host: test3.localhost` and ` Path:/test`are matched ;)

**Comment by emilevauge** [comment]
One minor comment ;)
Other than that, LGTM! This is really cool, thanks a lot @fclaeys :)
/cc @vdemeester 

Fixes #419

**Comment by fclaeys** [comment]
@emilevauge documentation OK

**Comment by vdemeester** [comment]
LGTM 🐮 🙌

**Pull request merged**